RadarSDK/RadarSettings.m

```diff
@@ -43,6 +45,23 @@ + (NSString *)installId {
     return installId;
 }
 
++ (NSString *)sessionId {
+    return [NSString stringWithFormat:@"%.f", [[NSUserDefaults standardUserDefaults] doubleForKey:kSessionId]];
+}
+
++ (BOOL)updateSessionId {
+    double timestampSeconds = [[NSDate date] timeIntervalSince1970];
+    double sessionIdSeconds = [[NSUserDefaults standardUserDefaults] doubleForKey:kSessionId];
+    if (timestampSeconds - sessionIdSeconds > 300) {
```

Comment:
nice, so we guard against the quick / unintentional backgrounding and then foregrounding

Reply:
Yep